.travis.yml

```diff
@@ -1,16 +1,12 @@
 language: python
 python:
  - 2.7
+ - 3.5
 sudo: false
-env:
- - TOX_ENV=py27-unittest
- - TOX_ENV=py27-integration
- - TOX_ENV=lint
- #- TOX_ENV=docs
 install:
- - pip install tox
+ - pip install tox-travis
```

Comment:
Does this not allow local testing? -1 from me on moving this to travis if so. I much prefer local debugging.